PixaPicker/JSONHandler.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  JSONHandler.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 9/26/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+class JSONHandler {
```

Comment:
I would make the class name more specific since you know you are returning PixaBay JSON response.  I typically call these type of classes something like `PixaBayAPIService`